src/storage/src/hummock/state_store_v1.rs

```diff
@@ -336,10 +336,16 @@ impl HummockStorageV1 {
                             .sstable(sstable_info, &mut local_stats)
                             .in_span(Span::enter_with_local_parent("get_sstable"))
                             .await?;
-
-                        if hit_sstable_bloom_filter(sstable.value(), *prefix_hash, &mut local_stats)
-                        {
-                            sstables.push((*sstable_info).clone());
+                        for table_id in &sstable_info.table_ids {
```

Comment:
we can get `table_id` from `read_options` and only check the associated `bloom_filter`